app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2_defconfig

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+CONFIG_SOC_SERIES_RP2XXX=y
+CONFIG_SOC_RP2040=y
+CONFIG_BOARD_KBDFANS_TOFU65_V2=y
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=125000000
+
+# Enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable USB CDC ACM logging for debugging
+CONFIG_ZMK_USB_LOGGING=y
+
```

Comment:
Shouldn't be on by default:



```suggestion
# Enable USB CDC ACM logging for debugging
# CONFIG_ZMK_USB_LOGGING=y

```

Reply:
Oops. This one I committed locally, but missed pushing it